src/App.jsx

```diff
@@ -1,37 +1,36 @@
 import React, { useState, useMemo } from 'react';
-import './App.css';
-import {mockedCoursesList, mockedAuthorsList} from "./Constants";
-import Header from './components/Header/Header.jsx';
-import Courses from './components/Courses/Courses';
-import CreateCourse from './components/Create courses/CreateCourse';
-
+import { Container } from 'reactstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import {mockedCoursesList, mockedAuthorsList} from "./constants";
+import Header from './components/header/header.jsx';
+import Courses from './components/courses/courses';
+import CreateCourse from "./components/create-courses/create-course";
 
 function App() {
-	const [page, setPage] = useState('courses')
-
+	//common data
 	const [courses, setCourses] = useState(mockedCoursesList)
```

Comment:
Prepare courses for your needs => map authors id to names,

Reply:
Done